packages/p2p/src/stores/my-ads-store.js

```diff
@@ -332,17 +340,29 @@ export default class MyAdsStore extends BaseStore {
     }
 
     @action.bound
-    restrictLength = (e, handleChange) => {
+    restrictLength = (e, handleChange, max_characters = 15) => {
         // typing more than 15 characters will break the layout
         // max doesn't disable typing, so we will use this to restrict length
-        const max_characters = 15;
+        // const max_characters = 15;
```

Comment:
Can we remove this line if it's not used anymore?

Reply:
Yes. I will remove it